mysql-test/suite/galera/t/galera_fc_auto_evict.test

```diff
@@ -0,0 +1,51 @@
+# PXC-3760: Implement a feature to evict a lagging node from PXC
+#
+# Problem:
+# We want to evict a node if it slows down a whole cluster due to the flow control.
+#
+# Solution:
+# We introduce two variables: gcs.fc_auto_evict_window and gcs.fc_auto_evict_threshold.
+# If the node pauses in FC more then "threshold * window" in the last  window" seconds
+# frame, node will just abort.
+#
+# It's just a smoke test of the feature, because it's problematic to achieve required
+# time characteristics. For more granular testing a unit test is used.
+
+--source include/galera_cluster.inc
+
+--connection node_2
+--source include/expect_crash.inc
+# Triggering flow control logic taken from here:
+# https://www.percona.com/blog/monitoring-flow-control-percona-xtradb-cluster-5-6/
+SET GLOBAL wsrep_provider_options="gcs.fc_limit=1;gcs.fc_auto_evict_window=0.1";
+FLUSH TABLES WITH READ LOCK;
+
+--connection node_1
+CREATE TABLE t1(a INT);
```

Comment:
Please wrap inserting loop in `--disable_query_log` / `--enable_query_log` and add `--echo ...` informing about what is happening.

Reply:
Done